docs/about-this-guidebook/git-workflow.md

```diff
@@ -0,0 +1,90 @@
+
+- **The PR title:** By default it is the title of your last commit.
+- **Extended description:** This also draws from your last commit.
+- **The right sidebar:** You can assign your pull request to someone on the team (someone you think may want to review your edits).
+
+Most of the time you don't need to fill anything out, just click "Create pull request".
+
+Or, if you want to make other changes on the same branch and have it reviewed at the same time as the commit you just made, don't create the PR quite yet. Navigate to the branch you created in your first commit (eg <https://github.com/CivicActions/handbook/tree/[your-branch-name]>. You can also find this branch from the handbook homepage by clicking on [branches](https://github.com/CivicActions/handbook/branches). Then, edit another file (or the same file you just edited) and commit it. This time, GitHub will know that you want to commit to the same branch as you created during your first commit.
+
+Once you are done making commits to your branch, you can then create a PR from your branch to the master branch, which will include all changes.
+
+## The more advanced way: fork the handbook
+
+**NB: Currently, pull requests submitted from forks of the handbook will not properly interact with the GitLab CI pipeline and will fail their automated tests. Until further notice, the preferred contribution method is to create a branch in the CivicActions/handbook repository and submit a PR from there.**
+
+If you're familiar with the Git fork and pull request (PR) workflow, we definitely recommend forking this repo and making pull requests from your fork. This is the recommended way to contribute.
+
+The easiest way to get started, if you're not familiar with the command line or an existing git repo management tool, is by installing [GitHub Desktop](https://desktop.github.com/), and then learning how to keep your fork up to date with the latest, create commits, and submit PRs.
+
+You can use any other number of tools, including text editors (such as [Atom](https://atom.io/)) and the command line, or other GUI tools.
+
+Resources:
+
+- [Fork a Repo](https://help.github.com/articles/fork-a-repo/)
+- [Working with Forks](https://help.github.com/articles/working-with-forks/)
+- [Getting Started with GitHub Desktop](https://help.github.com/en/desktop/getting-started-with-github-desktop)
+- [Contributing to Projects with GitHub Desktop](https://help.github.com/desktop/guides/contributing/)
+
+## Assignment of pull requests
+
+Your PR will automatically be assigned to a team for a review, based on which part of the handbook you are editing. That automatic review assignment is explained in [docs-governance.md](docs-governance.md). If you think that certain individuals on the team may have particular subject matter expertise or authority on the topic, you can also assign them directly via the "Reviewers" section of the right hand sidebar.
```

Comment:
@ContentBrewmaster I think the link for `docs-governance.md` should be `guidebook-governance.md`.